### DISCUSSION TITLE
Rust binding

### DISCUSSION THREAD

**Discussion by mrjbom** [spaskalev/buddy_alloc#123] [General]
Hi.
I liked your allocator and decided to use your allocator in my Rust project, I made a wrapper for it. It might be useful to mention it somewhere about it, maybe someone else would like to use your allocator in their Rust project.
https://github.com/mrjbom/buddy_alloc_rs

**Comment by spaskalev** [comment]
I'm glad that the allocator is working for you, good job with the bindings!

Would you be interested in a native port ? I was considering it for a while but no one had asked for it so far. Perhaps this Xmas :)

**Reply by mrjbom** [reply]
I think it would be good to have a native port.
